### PR TITLE
CTCTRADERS-2084: Seed multiple arrivals

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: app/testOnly/services/TestDataXMLGenerator.scala [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    val xml = TestDataXMLGenerator.arrivalNotification(mrn.format, eori.format)
+
+    val movementMessage = MovementMessageWithStatus(dateTime, ArrivalNotification, xml, MessageStatus.SubmissionPending, 1)
+
+    Arrival(arrivalId,
+            ChannelType.web,
+            MovementReferenceNumber(mrn.format),
+            eori.format,
+            Initialized,
```
Comment (ericohwotu) [inline app/testOnly/services/TestDataXMLGenerator.scala:51]: When the data is inserted will there be a way to transition out of Initialised? Do we plan on using the test-support service with these users?

Comment (amaalali) [reply app/testOnly/services/TestDataXMLGenerator.scala:51]: > When the data is inserted will there be a way to transition out of Initialised?

That's a great spot Eric! No there is nothing that will interact with these to transition the state from Initialized after this data is seeded. I'll update this to be ArrivalSubmitted

> Do we plan on using the test-support service with these users?

No plans currently but we can use it on an ad-hoc basis going forward, which means that the status needs to be ArrivalSubmitted as you pointed out above

I'll get this fixed and updated asap

File: app/testOnly/services/TestDataGenerator.scala [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+private[services] class TestDataGenerator @Inject()(clock: Clock) {
+
+  def arrivalMovement(eori: SeedEori, mrn: SeedMrn, arrivalId: ArrivalId): Arrival = {
+
+    val dateTime = LocalDateTime.now(clock)
+
+    val xml = TestDataXMLGenerator.arrivalNotification(mrn.format, eori.format)
+
+    val movementMessage = MovementMessageWithStatus(dateTime, ArrivalNotification, xml, MessageStatus.SubmissionPending, 1)
```
Comment (ericohwotu) [inline app/testOnly/services/TestDataGenerator.scala:45]: Last thing i think status should be submitted

**Comment by ericohwotu** [comment]
I think we also need an prod config change to disable the feature flag

**Review by ericohwotu — APPROVED** [review]

**Pull request merged**